.nsprc

```diff
@@ -1,10 +1,8 @@
 {
   "exceptions": [
-    "https://nodesecurity.io/advisories/532",
-    "https://nodesecurity.io/advisories/157",
-    "https://nodesecurity.io/advisories/577",
-    "https://nodesecurity.io/advisories/654",
-    "https://nodesecurity.io/advisories/664",
-    "https://nodesecurity.io/advisories/678"
+    "https://npmjs.com/advisories/532",
+    "https://npmjs.com/advisories/577",
+    "https://npmjs.com/advisories/722",
+    "https://npmjs.com/advisories/725"
```

Comment:
Should this file be renamed if it's a custom format now? The data could even be inlined in the audit script with proper comments. This used to have comments on why the exceptions were there, and that was very helpful (e.g. dev server vulnerability is ok).

Reply:
Just one clarification: it _never_ had comments since the original format `.nsprc` was JSON too.

I want to keep this separated from the actual audit script since the implementation is useless detail for most customizers, but updating the file would cause conflicts.

How about renaming the file to `.auditrc` (audit runtime configuration). I was also thinking of using JavaScript instead of JSON, which would allow comments.

Reply:
The original version did have comments:

https://github.com/sharetribe/flex-template-web/commit/eb9c909939ec0b06c6dd044e832a57dfcd4cebe3#diff-03cb303c8d3085901cf5a9214888ad1d

They just changed the parser at some point.

Reply:
:)

Reply:
Github is quite bad as a chat platform :D

Anyway, good feedback. I was hoping that you would comment about this file. I was trying to decide between following previous format or using JS (due to comments).

Anyway, after this (confusingly async) chat, I'll rename this file: `.auditrc` and use JavaScript inside of it.